src/components/CompareArtworks.jsx

```diff
@@ -0,0 +1,101 @@
+import { useEffect, useState } from "react";
+import { fetchArtworkCountsByMaterialAndCentury } from "../services/apiService";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from "recharts";
+
+function ArtworkComparison() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true); // Loading state
+
+  useEffect(() => {
+    const materials = ["chalk", "ink", "pencil"];
+    const centuries = { 18: 1700, 19: 1800, 20: 1900, 21: 2000 };
+    const loadData = async () => {
```

Comment:
The use of async/await ensures clarity and readability in handling asynchronous operations.

Reply:
Thanks!